src/main/java/uk/gov/hmcts/reform/civil/service/documentmanagement/ClaimFormService.java

```diff
@@ -17,9 +28,49 @@
 @ComponentScan("uk.gov.hmcts.reform")
 public class ClaimFormService {
 
+    private final CivilDocumentStitchingService civilDocumentStitchingService;
+    private final GenerateClaimFormForSpecCallbackHandler generateClaimFormForSpecCallbackHandler;
+    private final Time time;
+    private final DeadlinesCalculator deadlinesCalculator;
+
     public CaseDocument uploadSealedDocument(
-         String authorisation, CaseData caseData) {
-        return sealedClaimFormGeneratorForSpec.generate(caseData, authorisation);
+        String authorisation, CaseData caseData) {
+        LocalDate issueDate = time.now().toLocalDate();
+        CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder().issueDate(issueDate)
+            .respondent1ResponseDeadline(
+                deadlinesCalculator.plus14DaysAt4pmDeadline(LocalDateTime.now()))
+            // .respondent1Represented(YES)
+            .claimDismissedDate(null);
+        CaseDocument sealClaimForm = sealedClaimFormGeneratorForSpec.generate(caseData, authorisation);
+
+        List<DocumentMetaData> documentMetaDataList = generateClaimFormForSpecCallbackHandler
```

Comment:
This code can also be moved to Util class as this is duplicate of what is there in GenerateClaimFormSpecCallbackHandler

Reply:
alright, will incorporate it.